internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/QualysScannerConstants.java

```diff
@@ -37,6 +37,8 @@
     public static final String REPORT_UPLOAD_RETRY_INTERVAL = "scan_report_upload_retry_interval";
     public static final String SCHEDULER_DELAY = "schedulerDelay";
     public static final String SCAN_URL = "scanURL";
+    public static final String DEFAULT_REPORT_TEMPLATE_ID = "default_report_template_id";
+    public static final String REPORT_TEMPLATE_ID = "reportTemplateID";
```

Comment:
It looks wrong that we use two format here (camel-case and underscores).

IMO, we should name `REPORT_TEMPLATE_ID` as `PARAMETER_REPORT_TEMPLATE_ID` to show that it's a HTTP param. That will reduce the confusion.